class ChessException(Exception):
    destination = False

class ChessException_1(ChessException):
    num = 1

class ChessException_2(ChessException):
    num = 2

class ChessException_3(ChessException):
    num = 3

class ChessException_4(ChessException):
    num = 4

class ChessException_5(ChessException):
    num = 5

class ChessException_6(ChessException):
    num = 6

class ChessException_7(ChessException):
    num = 7

class ChessException_8(ChessException):
    num = 8

class ChessException_9(ChessException):
    num = 9
